files/es/web/api/console/timeend/index.md

```diff
@@ -29,7 +29,7 @@ console.timeEnd(etiqueta);
 
 ## Compatibilidad con Navegadores
 
-{{Compat("api.Console.timeEnd")}}
+{{Compat}}
```

Comment:
Como bien lo corriges es la manera correcta de usar la macro `{{Compat}}`
pero para que la macro funcione, necesita la referencia hacia el archivo donde tiene la información, dicha referencia debe de colocarse en `browser-compat` en la cabecera del documento.

Agrego el enlace hacia el mismo documento, donde estan usando el `browser-compat`

https://github.com/mdn/content/blob/main/files/en-us/web/api/console/timeend/index.md?plain=1#L13

y esto aplica a todos los documentos que actualizaste, saludos!

Reply:
El contenido localizado sin `browser-compat` será referenciado desde en-us.  Así que solo un cambio a Compat funciona aquí. 
Vista previa de la pàgina: https://pr10110.content.dev.mdn.mozit.cloud/es/docs/Web/API/Console/timeEnd

----
Localized content without `browser-compat` will be referenced from en-us.  So only a change to Compat works here.
Preview: https://pr10110.content.dev.mdn.mozit.cloud/es/docs/Web/API/Console/timeEnd

Reply:
Genial! Gracias